qiskit_experiments/analysis/plotting.py

```diff
@@ -40,7 +40,14 @@ def plot_curve_fit(
     Args:
         func: the fit function for curve_fit.
         result: an AnalysisResult from curve_fit.
-        confidence_interval: if True plot the confidence interval from popt_err.
+        monotonicity: a list or dictionary of Booleans, whose length is the number of parameters.
+           Choice of list or dictionary has to match the format of `result["popt"]`.
+           For every parameter, does the function increase when the parameter increases?
+           If not None, then a confidence interval will be displayed -
```

Comment:
Why is the existence of the confidence interval dependent on being given the monotonicity data?

Reply:
See here: https://github.com/Qiskit/qiskit-experiments/pull/149#discussion_r663510663